src/utils.js

```diff
@@ -47,3 +47,21 @@ export function limit(min, max, value) {
     }
     return value;
 }
+
+function getCSSPixelValue(stringValue) {
+    if (stringValue.substr(-2) === 'px') {
+        return parseFloat(stringValue);
+    }
+    return 0;
+}
```

Comment:
It's just a safe garde to ensure that `stringValue` is a pixels value since as mentioned on [StackOverfow](http://stackoverflow.com/questions/14275304/how-to-get-margin-value-of-a-div-in-original-javascript#answer-14275343):
> Fair warning: What you get back may not be in pixels. For instance, if I run the above on a p element in IE9, I get back "1em".

If you find that such browsers are not supported, I can remove this function and `parseFloat` directly.